conformance/lang/expressions/multiplicative-expr/division.balt

```diff
@@ -0,0 +1,4506 @@
+     b = 0.9E-10d;
+     c = a / b;
+     io:println(c); // @output -27777777777777777777777777777.77778
+     io:println(-25.E17d / 0.9E-10d); // @output -27777777777777777777777777777.77778
+
+     a = 17.E12d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output Infinity
+     io:println(17.E12d / 0.0d); // @output Infinity
+
+     a = -17.E12d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output -Infinity
+     io:println(-17.E12d / 0.0d); // @output -Infinity
+
+     a = 11.E12d;
+     b = -0.0d;
+     c = a / b;
+     io:println(c); // @output Infinity
+     io:println(11.E12d / -0.0d); // @output Infinity
+
+     a = 17.E1290d;
+     b = 13.E1521d;
+     c = a / b;
+     io:println(c); // @output 1.307692307692307692307692307692308E-231
+     io:println(17.E1290d / 13.E1521d); // @output 1.307692307692307692307692307692308E-231
+
+     a = 0.0d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output NaN
+     io:println(0.0d / 0.0d); // @output NaN
+
+     a = -0.0d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output NaN
+     io:println(-0.0d / 0.0d); // @output NaN
+}
+
+Test-Case: output
+Description: Test division scenarios for different nillable decimal values.
+Labels: multiplicative-expr, numeric-literal, float-point-literal
+
+public function main() {
+     decimal? a = 2.5E-17d;
+     decimal? b = 2.5E+17d;
+     decimal? c = b / a;
+     io:println(c); // @output 1E+34
+
+     a = 1.423223E6d;
+     b = 2.34413E2d;
+     c = a / b;
+     io:println(c); // @output 6071.433751541083472333019073174269
+
+     a = 25.E1742d;
+     b = 1.0d;
+     c = a / b;
+     io:println(c); // @output 2.5E+1743
+
+     a = 25.E1742d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output Infinity
+
+     a = 25.E17d;
+     b = 0.9E-10d;
+     c = a / b;
+     io:println(c); // @output 27777777777777777777777777777.77778
+
+     a = -25.E17d;
+     b = 0.9E-10d;
+     c = a / b;
+     io:println(c); // @output -27777777777777777777777777777.77778
+
+     a = 17.E12d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output Infinity
+
+     a = -17.E12d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output -Infinity
+
+     a = 11.E12d;
+     b = -0.0d;
+     c = a / b;
+     io:println(c); // @output Infinity
+
+     a = 17.E1290d;
+     b = 13.E1521d;
+     c = a / b;
+     io:println(c); // @output 1.307692307692307692307692307692308E-231
+
+     a = 0.0d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output NaN
+
+     a = -0.0d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output NaN
+
+     a = 17.E1290d;
+     b = ();
+     c = a / b;
+     io:println(c.toBalString()); // @output ()
+
+     a = ();
+     b = 13.E1521d;
+     c = a / b;
+     io:println(c.toBalString()); // @output ()
+}
+
+Test-Case: output
+Description: Test division scenarios for different float values.
+Labels: multiplicative-expr, numeric-literal, float-point-literal
```

Comment:
Shall we use `DecimalFloatingPointNumber` instead of `floating-point-literal`?
Don't need `numeric-literal` label? 
Missing `float`, `FloatTypeSuffix`, `unary-minus` labels.